widget.subscribe.php

```diff
@@ -359,33 +359,8 @@ function add_subscriber() {
 				}
 			} //end for loop
 
-
-			$subscribe_to_lists  = [];
-			if ( isset($_POST['sailthru_email_list']) ){
-				$sailthru_email_list = sanitize_text_field( $_POST['sailthru_email_list'] );
-			}
-			if ( ! empty( $sailthru_email_list ) ) {
-
-				// check for double opt in setting
-				if ( isset( $customfields['sailthru_double_opt_in'] ) && true === $customfields['sailthru_double_opt_in'] ) {
-					$double_opt_in = true;
-				} else {
-					$double_opt_in = false;
-				}
-
-				$lists = explode( ',', $sailthru_email_list );
-
-				foreach ( $lists as $key => $list ) {
-					$subscribe_to_lists[ $list ] = 1;
-				}
-
-				$options['lists'] = $subscribe_to_lists;
-
-			} else {
-
-				$options['lists'] = array( 'Sailthru Subscribe Widget' => 1 ); // subscriber is an orphan
-
-			}
+            $double_opt_in = !empty( $customfields['sailthru_double_opt_in'] );
```

Comment:
This will check against the value in the sailthru_double_opt_in. I actually tested all possibilities that I thought about, anything other than $customfields['sailthru_double_opt_in] equals to 1 or true will evaluate to false. Here are the results:

`!empty( $customfields['sailthru_double_opt_in'] ):

undefined $customfields : false
$customfields = []: false
$customfields['sailthru_double_opt_in'] = null: false
$customfields['sailthru_double_opt_in'] = '': false
$customfields['sailthru_double_opt_in'] = 0: false
$customfields['sailthru_double_opt_in'] = 1: true
$customfields['sailthru_double_opt_in'] = 0: false
$customfields['sailthru_double_opt_in'] = 0: false
$customfields['sailthru_double_opt_in'] = false: false
$customfields['sailthru_double_opt_in'] = true: true`